src/main/java/com/minecolonies/entity/EntityFishHook.java

```diff
@@ -736,6 +736,7 @@ private void spawnLootAndExp(final EntityCitizen citizen)
      * @param citizen the fisherman getting the loot
      * @return an ItemStack randomly from the loot table
      */
+    @SuppressWarnings("unchecked")
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Suppressing warnings is not allowed [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1309)